common/src/main/scala/uk/ac/wellcome/models/Agent.scala

```diff
@@ -13,6 +13,6 @@ case class Organisation(
 ) extends AbstractAgent
 
 case class Person(label: String,
-                  prefix: Option[String] = None,
+                  prefixes: Option[List[String]] = None,
```

Comment:
I think I remember having a discussion with @jtweed or @silveroliver about this and them deciding towards a simple concatenated string. Wasn't reflected in the example file though I think. Is that changed now?